demo/demo.cpp

```diff
@@ -14,7 +14,7 @@
 
 #include "model_loader_assimp.hpp"
 
-#define SCENE viknell_scene
+#define SCENE emibl_scene
```

Comment:
revert this. When reverting to old commits we want the test scene to be the same so we can detect visual errors without having to adjust the scene everytime.